docs/BR.md

```diff
@@ -1099,8 +1103,40 @@ No stipulation.
 
 ### 4.3.1 CA actions during certificate issuance
 
+#### 4.3.1.1 Manual authorization of certificate issuance for Root CAs
+
 Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
 
+#### 4.3.1.2 Linting of to-be-signed Certificate content
+
+Due to the complexity in implementing the certificate-formatting standards of these Requirements when issuing publicly-trusted Certificates, before a Certificate or Precertificate [RFC 6962] is issued, effective 2024-09-15 the CA SHOULD implement a Linting process to test the technical conformity of the to-be-signed Certificate with these Requirements. Effective 2025-03-15, the CA SHALL implement a Linting process to test the technical conformity of the to-be-signed Certificate with these Requirements.
+
+Methods used to produce a certificate containing the to-be-signed Certificate content include, but are not limited to:
+
+1. Sign the `tbsCertificate` with a "dummy" Private Key whose Public Key component is not certified by a Certificate that chains to a publicly-trusted CA Certificate; or
+2. Specify a static value for the `signature` field of the Certificate ASN.1 SEQUENCE.
+
+CAs MAY implement their own certificate Linting tools, but CAs SHOULD use the Linting tools that have been widely adopted by the industry (see https://cabforum.org/resources/tools/). 
+
+CAs are encouraged to contribute to open-source linting projects, such as by:
+
+- creating new or improving existing lints,
+- reporting Requirements that are not covered by existing lints ("unlinted issues"), 
+- updating documentation of existing lints, and 
+- generating test certificates for positive/negative tests of specific lints.
+
+#### 4.3.1.3 Linting of issued Certificates
+
+CAs MAY use a Linting process to test each issued Certificate.
+
+CAs SHOULD use a Linting process to test the technical accuracy of already issued Certificates against the sample set described in [section 8.7](#87-self-audits).
+
+#### 4.3.1.4 Updates of Linting software
```

Comment:
I think this should be a sub-section of 4.3.1.2, as it is not an action taken by the CA prior to Certificate issuance.

Reply:
Perhaps the update process of a linter should go to another section (perhaps the currently-empty section 6.6.1 System development controls), and perhaps the suggestion for linting final certificates should go to section 8.7.

Thoughts about that?

Reply:
Addressed in 333693934193c733ec5a7f775029ef033c6db5c4